### ISSUE TITLE
[Health 10.1.0] Android build fails (was working in 10.0.0)

### ISSUE THREAD

**Issue by ciriousjoker** [cph-cachet/flutter-plugins#934]
**TLDR:**
https://github.com/cph-cachet/flutter-plugins/blob/b62d2e544eee27eea90466b9da82ed415f4eaa79/packages/health/android/src/main/kotlin/cachet/plugins/health/HealthPlugin.kt#L3005

That "w" was probably a typo and breaks compilation.

---

### Device / Emulator and OS
Please complete the following information for each phone and/or emulator you're experiencing this bug on:
 - Device: Pixel 7 Pro API 34 Emulator
 - OS: Android 14

**NB: Bugs pertaining to old devices/OS versions will likely not be fixed.**
* For iOS only the latest OS is supported. 
* For Android, see [the OS versions for which Google support security fixes](https://en.wikipedia.org/wiki/Android_version_history)**

### Describe the bug

Android build fails.

### To Reproduce

Update plugin version from 10.0.0 to 10.1.0.

### Expected behavior

Should work as it did in 10.0.0.

### Actual behavior

> e: file:///Volumes/Storage/Cache/pub/hosted/pub.dev/health-10.1.0/android/src/main/kotlin/cachet/plugins/health/HealthPlugin.kt:3005:77 Expecting ','
> e: file:///Volumes/Storage/Cache/pub/hosted/pub.dev/health-10.1.0/android/src/main/kotlin/cachet/plugins/health/HealthPlugin.kt:3005:76 Unresolved reference: w
> 
> FAILURE: Build failed with an exception.
> 
> * What went wrong:
> Execution failed for task ':health:compileDebugKotlin'.
> > A failure occurred while executing org.jetbrains.kotlin.compilerRunner.GradleCompilerRunnerWithWorkers$GradleKotlinCompilerWorkAction
>    > Compilation error. See log for more details
> 
> * Try:
> > Run with --stacktrace option to get the stack trace.
> > Run with --info or --debug option to get more log output.
> > Run with --scan to get full insights.
> 
> * Get more help at https://help.gradle.org
> 
> BUILD FAILED in 37s
> Error: Gradle task assembleDebug failed with exit code 1

### Flutter doctor 

> Doctor summary (to see all details, run flutter doctor -v):
> [✓] Flutter (Channel stable, 3.19.1, on macOS 14.4.1 23E224 darwin-arm64, locale de-DE)
> [✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
> [✓] Xcode - develop for iOS and macOS (Xcode 15.1)
> [✓] Chrome - develop for the web
> [✓] Android Studio (version 2023.2)
> [✓] VS Code (version 1.87.2)
> [✓] Connected device (2 available)
> [✓] Network resources
> 
> • No issues found!

### Additional information

This issue doesn't exist in 10.0.0, caused by https://github.com/cph-cachet/flutter-plugins/pull/926.

**Comment by bardram** [comment]
👍  - released a fix in 10.1.1
